.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: required
+dist: trusty
+language: node_js
+addons:
+  firefox: 'latest'
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+node_js:
+  - '8'
+
+install:
+  - npm install
+
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+script:
+  - $CI_ACTION
+
+env:
+  matrix:
+    - CI_ACTION="npm test -- --browsers Firefox --single-run"
+    - CI_ACTION="npm test -- --browsers Chrome --single-run"
+    - CI_ACTION="npm run lint"
+    - CI_ACTION="npm run dist"
+
+branches:
+  only:
+    - travis
+
+cache:
+  directories:
+    - node_modules
+
```

Comment:
Travis config is mostly a copy of [aframe's config](https://github.com/aframevr/aframe/blob/master/.travis.yml).